bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/ShellyBindingConstants.java

```diff
@@ -250,9 +250,8 @@
     public static final String CHANNEL_BUTTON_TRIGGER = "button";
 
     public static final String SERVICE_TYPE = "_http._tcp.local.";
-    public static final String SHELLY_API_MIN_FWVERSION = "v1.5.7";// v1.5.2
-    public static final String SHELLY_API_MIN_FWCOIOT_STD = "v1.6";// v1.5.2
-    public static final String SHELLY_API_MIN_FWCOIOT_SENSOR = "v1.7";// v1.5.2
```

Comment:
What happened to this version?

Reply:
I don't know what's displayed here. This is the latest code:
```
    public static final String SERVICE_TYPE = "_http._tcp.local.";
    public static final String SHELLY_API_MIN_FWVERSION = "v1.5.7";// v1.5.7+
    public static final String SHELLY_API_MIN_FWCOIOT = "v1.6";// v1.6.0+
```